contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,200 @@
+<frontmatter>
+  title: Introduction to Angular
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Angular
+
+**Author: [Ronak Lakhotia](https://github.com/RonakLakhotia)**                                                              
+Reviewers: [Tan Heng Yeow](https://github.com/tanhengyeow), [Jacob Li PengCheng](https://github.com/jacoblipech)
+
+## What is Angular?
+
+Angular, according to the [official website](https://angular.io/docs), is described as follows:
+
+>Angular is a platform that makes it easy to build applications with the web. Angular combines declarative templates, dependency injection, end to end tooling, and integrated best practices to solve development challenges. Angular empowers developers to build applications that live on the web, mobile, or the desktop.
+
+Let us understand some key terms starting with what <i>declarative templates</i> mean.
+Consider the example of creating a simple web application as shown below.
+
+![Example](First_example.png)
+
+The application displays the details of a person, that is the id and the name. The view component (i.e., the part users see) is defined in Angular using templates. These templates are nothing but HTML files. Additionally, Angular extends the vocabulary of HTML by allowing you to create your own custom tags.
```

Comment:
Does the example contain any custom tags?

In any case, the custom tag explanation seems an extra thing that slows down the reader. You can rephrase as follows and give a tooltip for `custom tags`:

An Angular template contains HTML code and custom tags.